stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageList.kt

```diff
@@ -91,6 +92,9 @@ public fun MessageList(
             viewModel.focusMessage(it.messageId)
         }
     },
+    onModeratedItemInteraction: (Message) -> Unit = {
+        viewModel.selectModeratedMessage(it)
```

Comment:
We could just have all the logic in selectMessage and then decide what to do there, rather than have multiple functions that kind of do the same thing